packages/core/src/panel/Panel.css

```diff
@@ -11,7 +11,8 @@
 /* Styles applied to the root element */
 .saltPanel {
   background: var(--saltPanel-background, var(--panel-background));
-  color: var(--saltPanel-color, initial);
+  color: var(--saltPanel-color, inherit);
+  font-family: var(--saltPanel-fontFamily, var(--salt-text-fontFamily));
```

Comment:
I believe this has already been tried and were previously removed https://github.com/jpmorganchase/salt-ds/pull/2952
cc @origami-z

Reply:
issue seems to be related to font family but I think color fix would be suitable here? @origami-z

Reply:
If `color` change fixes the issue, could do. Need to do a few tests across browsers to understand impact